### PR TITLE
fixed the filter_date issue

### PR DESCRIPTION
**Pull request by athiruma**
Fixed the filter_date issue and removed cloud from AWS Cloud

### CODE REVIEW

File: cloud_governance/policy/ibm/cost_billing_reports.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def __init__(self):
     def prepare_es_data(self, usage_cost: float, next_invoice: float):
         """This method prepares the data to upload to the es"""
         date = datetime.datetime.now()
-        start_date = f'{date.year}-{date.month}-01'
+        start_date = f'{date.year}-{date.strftime("%m")}-01'
```
Comment (ebattat) [inline cloud_governance/policy/ibm/cost_billing_reports.py:31]: Why we need 01, I think that month and year is enough, WDYT ?

**Comment by ebattat** [comment]
@athiruma, Did you fix IBM date format issue ?

File: cloud_governance/policy/ibm/cost_billing_reports.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/policy/ibm/cost_billing_reports.py:31]: We need to select always start_day of the month. So I included the 01

**Comment by athiruma** [comment]
> @athiruma, Did you fix IBM date format issue ?

yes

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**